instance/models/openedx_appserver.py

```diff
@@ -144,6 +154,8 @@ def set_field_defaults(self):
         """
         Set default values.
         """
+        if not self.github_admin_organizations and settings.DEFAULT_ADMIN_ORGANIZATION:
```

Comment:
We will almost always want `DEFAULT_ADMIN_ORGANIZATION`to be included on the list no matter what, so I think the way it is now is preferable.